Source/Plugins/Core/com.equella.core/src/com/tle/core/freetext/service/FreeTextService.java

```diff
@@ -43,6 +43,18 @@
   <T extends FreetextResult> FreetextSearchResults<T> search(
       Search searchreq, int nStart, int nCount);
 
+  /**
+   * Another search method which allows users to specify if they want to search attachments.
+   *
+   * @param searchReq An search request
+   * @param start The first record of a search result
+   * @param count The number of records of a search result
+   * @param searchAttachments Whether to search attachments
+   * @return An instance of SearchResults
```

Comment:
The method signature already tells us this, so how about:

```suggestion
   * @return Search results matching the specified criteria, with a maximum length of count.
```